api/api.proto

```diff
@@ -907,35 +900,33 @@ service OpenStorageCredentials {
   rpc EnumerateForAWS(SdkCredentialEnumerateAWSRequest)
     returns (SdkCredentialEnumerateAWSResponse) {
       option(google.api.http) = {
-        post: "/v1/credentials/enumerate/aws"
-        body: "*"
+        get: "/v1/credentials/aws"
```

Comment:
Same here, probably need one API to enumerate all credentials instead of having separate ones.

Reply:
That is a great point. Created #469 to fix this.